content/en/docs/collector/custom-collector.md

```diff
@@ -0,0 +1,161 @@
+
+| Tag              | Description                                                                                        | Optional | Default Value                                                                     |
+| ---------------- | -------------------------------------------------------------------------------------------------- | -------- | --------------------------------------------------------------------------------- |
+| module:          | The module name for the new distribution, following Go mod conventions. Optional, but recommended. | Yes      | "go.opentelemetry.io/collector/cmd/builder"                                       |
+| name:            | The binary name for your distribution                                                              | Yes      | "otelcol-custom"                                                                  |
+| description:     | A long name for the application.                                                                   | Yes      | "Custom OpenTelemetry Collector distribution"                                     |
+| otelcol_version: | The OpenTelemetry Collector version to use as base for the distribution.                           | Yes      | "0.53.0"                                                                          |
+| output_path:     | The path to write the output (sources and binary).                                                 | Yes      | "/var/folders/86/s7l1czb16g124tng0d7wyrtw0000gn/T/otelcol-distribution3618633831" |
+| version:         | The version for your custom OpenTelemetry Collector.                                               | Yes      | "1.0.0"                                                                           |
+| go:              | Which Go binary to use to compile the generated sources.                                           | Yes      | go from the PATH                                                                  |
+
+As you can see on the table above, all the `dist` tags are optional, so you will
+be adding custom values for them depending if your intentions to make your
+custom Collector distribution available for consumption by other users or if you
+are simply leveraging the `ocb` to bootstrap your component development and
+testing environment.
+
+For this tutorial, you will be creating a Collector's distribution to support
+the development and testing of components.
+
+Go ahead and create a manifest file named `builder-config.yaml` with the
+following content:
+
+> builder-config.yaml
+
+```yaml
+dist:
+  name: otelcol-dev
+  description: "Basic OTel Collector distribution for Developers"
+  output_path: ./otelcol-dev
+```
+
+Now you need to add the modules representing the components you want to be
+incorporated in this custom Collector distribution. Take a look at the
+[ocb configuration documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder#configuration)
```

Comment:
move this url to a reference URL at the bottom?

Reply:
humm it's in context though, so I wouldn't move it.

Reply:
I _think_ that @svrnm was asking that this be made into a markdown link definition. But this is ok for now IMHO.